utils/SophiaUtil.js

```diff
@@ -0,0 +1,98 @@
+const utils = require('./../cli-commands/utils');
+
+// TODO: insert and use 'Assert' 
+// const assertInterface = `
+// contract Assert =
+//     public function require : (bool,string) => ()
+
+
+// `;
+
+class SophiaUtil {
```

Comment:
please rename the file with lower case and dashes, to be consistent with the other files